examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -194,8 +194,9 @@ function default_cache(
         Yₜ = similar(Y), # only needed when using increment formulation
         limiters,
         ᶜρh_kwargs...,
-        ᶜuvw = similar(Y.c, Geometry.Covariant123Vector{FT}),
         ᶜK = similar(Y.c, FT),
+        ᶠũ = similar(Y.f, Geometry.Contravariant123Vector{FT}),
+        ᶜū = similar(Y.c, Geometry.Covariant123Vector{FT}),
```

Comment:
Might be easier to read the more verbose form:
```suggestion
        ᶠu_tilde = similar(Y.f, Geometry.Contravariant123Vector{FT}),
        ᶜu_bar = similar(Y.c, Geometry.Covariant123Vector{FT}),
```